app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -242,14 +239,9 @@ $after-button-padding-left: govuk-spacing(5);
     }
 
     @include govuk-media-query($from: "desktop") {
-      float: left;
-      font-size: 16px;
-
       // stylelint-disable max-nesting-depth
```

Comment:
Is this still needed, or was this only here to allow the extra level of nesting required for the `@if`?

Reply:
Just tested and yes, it's still needed.